### PR TITLE
build(deps): bump @keplr-wallet/types from 0.10.19 to 0.11.17

### PR DESCRIPTION
**Pull request by dadamu**
## Description
This PR bumps  @keplr-wallet/types from 0.10.19 to 0.11.17 and fix the break change.
Forwarding: #91 
Closes: #91

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/desmos-labs/desmjs/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] confirmed that this PR does not change production code

### CODE REVIEW

File: packages/keplr/src/chains.ts [hunk 1]
```diff
@@ -26,7 +25,6 @@ export async function setupChainInfo(
     currencies: chain.currencies,
     feeCurrencies: chain.feeCurrencies,
```
Comment (manu0466) [inline packages/keplr/src/chains.ts:26]:
```suggestion
    feeCurrencies: {
      ...chain.feeCurrencies,
      gasPriceStep: DesmosGasPriceStep,
    },
```
The `gasPriceStep` has been moved inside the `feeCurrencies` field you can see the doc [here](https://docs.keplr.app/api/suggest-chain.html).

File: packages/keplr/src/chains.ts [hunk 2]
```diff
@@ -2,7 +2,6 @@ import { ChainInfo } from "@keplr-wallet/types";
 import {
   ChainInfo as DesmJSChainInfo,
   DesmosBech32Config,
-  DesmosGasPriceStep,
```
Comment (manu0466) [inline packages/keplr/src/chains.ts:5]: Remember to reimport `DesmosGasPriceStep`

**Review by manu0466 — CHANGES_REQUESTED** [review]

**Review by manu0466 — APPROVED** [review]
LGTM

**Review by RiccardoM — APPROVED** [review]
utACK

**Pull request merged**